openmeetings-web/src/main/webapp/css/raw-room.css

```
@@ -28,37 +16,44 @@
 	padding: 0;
 	margin: 0;
 }
-.room-block .sidebar {
+.room-block .sb-wb {
+	height: calc(100% - var(--room-menu-height));
+	width: calc(100% - var(--chat-width) - var(--buffer-size));
+	display: inline-block;
+	position: relative;
+}
+.room-block .sb-wb .sidebar {
 	min-width: 155px;
 	width: var(--room-sidebar-width);
 	position: absolute;
-	height: calc(100% - var(--room-menu-height));
+	height: 100%;
+	top: 0;
 	right: auto;
 	left: 0;
 }
-.room-block .sidebar #room-sidebar-tabs {
+.room-block .sb-wb .sidebar #room-sidebar-tabs {
 	height: calc(100% - var(--buffer-size) - var(--activities-height));
 }
-.room-block .sidebar .tab {
+.room-block .sb-wb .sidebar .tab {
 	padding: 5px 0 0 0;
 }
-.room-block .sidebar .tab i {
+.room-block .sb-wb .sidebar .tab i {
 	font-size: 2.0em;
 	vertical-align: top;
 }
-html[dir="rtl"] .room-block .sidebar {
+html[dir="rtl"] .room-block .sb-wb .sidebar {
 	right: 0;
 	left: auto;
 }
-.room-block .sidebar #busy-indicator {
+.room-block .sb-wb .sidebar #busy-indicator {
```

Same, can it not just be:
```suggestion
#busy-indicator {
```